integration_test/ops_agent_test.go

```diff
@@ -1625,6 +1632,128 @@ func TestWindowsEventLog(t *testing.T) {
 	})
 }
 
+func TestWindowsEventLogV1UnsupportedChannel(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		t.Parallel()
+		if !gce.IsWindows(platform) {
+			t.SkipNow()
+		}
+		ctx, logger, vm := agents.CommonSetup(t, platform)
+
+		log := "windows_event_log"
+		channel := "Microsoft-Windows-User Control Panel/Operational"
+		config := fmt.Sprintf(`logging:
+  receivers:
+    %s:
+      type: windows_event_log
+      channels:
+      - Application
+      - %s
+  service:
+    pipelines:
+      default_pipeline:
+        receivers: [%s]
+`, log, channel, log)
+		if err := setupOpsAgent(ctx, logger, vm, config); err != nil {
+			t.Fatal(err)
+		}
+
+		// Quote-and-escape the query string so that Cloud Logging accepts it
+		expectedWarning := fmt.Sprintf(`"\"logging.receivers.%s.channels\" contains a channel, \"%s\", which may not work properly on version 1 of windows_event_log"`, log, channel)
+		if err := gce.WaitForLog(ctx, logger.ToMainLog(), vm, log, time.Hour, logMessageQueryForPlatform(vm.Platform, expectedWarning)); err != nil {
+			t.Fatal(err)
+		}
+	})
+}
+
+func TestWindowsEventLogV2(t *testing.T) {
```

Comment:
What will the error message be like when winlog is dealing with a non-exstent channel, e.g. a typo when they input the channel name.

Reply:
For v1, fluent-bit silently accepts the mis-typed channel but treats it as `Application` so you end up with duplicate logs.

For v2, fluent-bit emits an error and then terminates. These errors aren't surfaced to Cloud Logging but would be included in the diagnostics bundle.

```
[2023/02/15 10:27:52] [error] [in_winevtlog] cannot subscribe 'Applicatio' (0)
[2023/02/15 10:27:52] [error] [lib] backend failed
[2023/02/15 10:27:52] [error] [input:winevtlog:winevtlog.1] failed to open channels
[2023/02/15 10:27:52] [error] Failed initialize input winevtlog.1
```